src/controllers/update/update.director.details.controller.ts

```diff
@@ -19,6 +20,8 @@ export const get = async (req: Request, resp: Response, next: NextFunction) => {
     const session: Session = req.session as Session;
     const companyProfile: CompanyProfile = await getCompanyProfile(companyNumber);
     const officerFiling: OfficerFiling = await getOfficerFiling(session, transactionId, submissionId);
+    const locales = getLocalesService();
+    const lang = selectLang(req.query.lang);
     let updatedData = false;
 
```

Comment:
Seems currentUrl is missing (needed for localisation toggle). Missed it out for active directors too

Reply:
Thanks, have added currentUrl to both update.director.details and active.directors